backend/django/core/templates/projects/create/create_wizard_labels.html

```diff
@@ -104,25 +112,29 @@ <h3>Instructions</h3>
           var csv = event.target.result;
           var data = $.csv.toArrays(csv);
           if(data[0][0] === 'Label' && data[0][1] === 'Description') {
-            data.forEach((label, index) => {
-              if(index !== 0) {
-                if($('.formset_row_label').eq(index - 1).length > 0) {
-                  $('.formset_row_label').eq(index - 1).css('display', '');
-                  $('.formset_row_label').eq(index - 1).find('input[type=text]').eq(0).val(label[0]);
-                  if(label[1].length > 0) {
-                    $('.formset_row_label').eq(index - 1).find('textarea').eq(0).val(label[1]);
-                  } else {
-                    $('.formset_row_label').eq(index - 1).find('textarea').eq(0).val('');
-                  }
-                } else {
-                  $(':contains("add label")').click();
-                  $('.formset_row_label').last().find('input[type=text]').eq(0).val(label[0]);
-                  if(label[1].length > 0) {
-                    $('.formset_row_label').last().find('textarea').eq(0).val(label[1]);
-                  }
-                }
-              }
-            });
+            data.shift();
+            let appendHTML = data.map((label, index) =>
+              `<tr class="formset_row_label dynamic-form">
+                <td>
+                  <input type="hidden" name="label_set-${index}-project" id="id_label_set-${index}-project" value=""/></input>
+                  <input type="hidden" name="label_set-${index}-id" id="id_label_set-${index}-id" value=""></input>
+                  <div class="error-messages"></div>
+                  <input type="text" name="label_set-${index}-name" class="form-control" id="id_label_set-${index}-name" value="${label[0]}"></input>
+                </td>
+                <td>
+                  <div class="error-messages"></div>
+                  <textarea name="label_set-${index}-description" cols="40" rows="5" class="form-control" id="id_label_set-${index}-description" value="${label[1]}">${label[1]}</textarea>
+                </td>
+                <td>
+                  <div class="error-messages"></div>
+                  <a class="delete-row" href="javascript:void(0)" style="">remove label</a></td>
```

Comment:
So when I use this to add a bunch of labels, the remove label button is no longer working

Reply:
I wrote a new function to handle deleting labels when a csv is used, hopefully that fixes it!